.github/workflows/build.yaml

```diff
@@ -186,7 +186,7 @@ jobs:
         uses: ./.github/actions/parallel-ctest-containers
         with:
           container: ${{fromJSON(needs.d.outputs.p)[matrix.platform].image}}
-          error-log-paths: '["build/etc", "build/var", "build/leap-ignition-wd"]'
+          error-log-paths: '["build/etc", "build/var", "build/leap-ignition-wd", "build/TestLogs/*"]'
```

Comment:
I think it should just be `"build/TestLogs"`. The code just looks at `startsWith()` on these

https://github.com/AntelopeIO/leap/blob/e1b51aa0f34f86e190c6e50d5d0bbfb6b6ba044b/.github/actions/parallel-ctest-containers/main.mjs#L54-L58